src/block/edit.js

```diff
@@ -74,7 +74,7 @@ class ButtonEdit extends Component {
 
 		return (
 			<Fragment>
-				<div className={ className, classnames( 'eig_call' ) } title={ title } ref={ this.bindRef }>
```

Comment:
Had to remove this in order for className to work properly. Would love to know how I can add this class back in though?